orcid-core/src/main/java/org/orcid/core/solr/OrcidSolrOrgsClient.java

```diff
@@ -56,13 +56,13 @@ public OrgDisambiguatedSolrDocument findById(Long id) {
         SolrQuery query = new SolrQuery();
         query.setQuery(queryString.toString());
         query.addOrUpdateSort("score", ORDER.desc);
-        if (promoteChosenOrgs) {
+        /*if (promoteChosenOrgs) {
             query.addOrUpdateSort("org-chosen-by-member", ORDER.desc);
-        }
+        }*/
         query.addOrUpdateSort("org-disambiguated-popularity", ORDER.desc);
 
         query.addFilterQuery(String.format("(%s:(%s OR %s OR %s)) OR (%s:%s AND %s:%s)", SolrConstants.ORG_DISAMBIGUATED_ID_SOURCE_TYPE, "ROR", "RINGGOLD", "FUNDREF",
```

Comment:
@Camelia-Orcid I think you should fix the query and remove the last AND statement.
Since you removed the last two parameters: 

- SolrConstants.ORG_CHOSEN_BY_MEMBER
- true

Then, the last two replacement of the query are going to be missing. 

**This:** 

(%s:(%s OR %s OR %s)) OR (%s:%s AND %s:%s)

**Will translate to this:** 

(ORG_DISAMBIGUATED_ID_SOURCE_TYPE:(ROR OR RINGGOLD OR FUNDREF)) OR (ORG_DISAMBIGUATED_ID_SOURCE_TYPE:LEI AND **_%s:%s_**)

So, you should remove the last `AND %s:%s`